bin/oref0-setup.sh

```diff
@@ -703,8 +703,8 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
         )
     fi
 
+    cp preferences.json old_preferences.json
```

Comment:
I moved this line here so we can always have access to old_preferences. I was wondering if i should additionally date the file name?